CMakeLists.txt

```diff
@@ -54,6 +72,12 @@ link_directories(
   ${OFED_PREFIX}/lib
   ${LEVELDB_PREFIX}/lib
 )
+if(${FREEBSD})
```

Comment:
On 4-7-2016 16:20, Kefu Chai wrote:
> In CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10024#discussion_r69464221>:
> 
>> @@ -54,6 +72,12 @@ link_directories(
>>    ${OFED_PREFIX}/lib
>>    ${LEVELDB_PREFIX}/lib
>>  )
>> +if(${FREEBSD})
> 
> could you be specific what errors you are reference?

phew, that is hard.... don't really remember
And I'm not sure that it was with this variable.
But I was trying some conditional code that did not work, until I
wrapped it in |${}|

But I'm on my way to change all the WITH_* I created to |if(WITH_*)|,
and we'll see what comes of it.